<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>10 самых востребованных товаров за указанное время</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tw">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_date_FROM">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Начальная дата:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_date_BEFORE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Конечная дата:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_date_FROM">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2010-01-01</string>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_date_BEFORE">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2015-12-30</string>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="AcceptButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>470</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Послать
запрос!</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>300</y>
      <width>421</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="QVBL"/>
   </widget>
   <widget class="QGroupBox" name="gist_groupBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>460</y>
      <width>101</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Выберите вид:</string>
    </property>
    <widget class="QRadioButton" name="ALL_RB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Полный вид</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="TOP_RB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Только верх</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="criterium_groupBox">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Критерий выбора:</string>
    </property>
    <widget class="QRadioButton" name="Duty_RB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Пошлина</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="Weight_RB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Вес</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Amount_RB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Объём</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="update_groupBox">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>361</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Изменение данных о товаре</string>
    </property>
    <widget class="QGroupBox" name="upd_criterium_groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Изменяемая величина:</string>
     </property>
     <widget class="QRadioButton" name="upd_duty_RB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Пошлина</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="upd_weight_RB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Вес</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="upd_amount_RB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Количество</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="upd_volume_RB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Объём</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="new_data_LE">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="maxLength">
      <number>21</number>
     </property>
    </widget>
    <widget class="QLabel" name="new_data_LABEL">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Новое значение:</string>
     </property>
    </widget>
    <widget class="QLabel" name="criterion_id_LABEL">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>ID изменяемого критерия:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="upd_ID_LE">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="maxLength">
      <number>21</number>
     </property>
    </widget>
    <widget class="QPushButton" name="upd_button">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="show_DB">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>75</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Показать
текущие 
данные</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>844</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
